yellowbrick/draw.py

```diff
@@ -0,0 +1,82 @@
+# yellowbrick.draw
+# Utilities for common matplotlib drawing procedures.
```

Comment:
Love this idea of having a special module of drawing utilities! Now I'm wondering which other things we might want to go in here - maybe methods for annotating plots with the visualizer's scores or fit times, or other metadata like that?

Reply:
Sure, my feeling is that anywhere in our code where we do the same n number of matplotlib steps, might be a good candidate for a drawing utility.